portfolio/src/main/webapp/blogs.html

```diff
@@ -15,17 +15,22 @@ <h1>Kathy's Blog Posts</h1>
       <nav role="navigation" aria-label="navigation bar">
         <a href="#blog1">Blog 1</a><br>
         <a href="#blog2">Blog 2</a><br>
+        <a href="#blog3">Blog 3</a><br>
       </nav>
       <p>Welcome to my blog posts on hidden talents!</p>
 
       <h3 id="blog1">Hidden Talent NO.1</h3>
       <p>I know how to play badminton. I've always enjoyed the sport since I was a child, and I used to
           play it every week when I was in middle school. I haven't played it for a while now, so I'm quite rusty.
-          But badminton is still my favorite sport regardless.
-      </p>
+          But badminton is still my favorite sport regardless.</p>
 
       <h3 id="blog2">Hidden Talent NO.2</h3>
       <p>I've never had a nose bleed.</p>
+
+      <h3 id="blog3">Hidden Talent NO.3</h3>
+      <p>I just started learning JavaScript and <em>this</em> portfolio is my first piece of code.<br><br>
+          <b>Would you like to write some JavaScript as well? Here is a mini-console:</b></p>
+      <button onclick="executeJavaScript()">SURE!</button>
```

Comment:
Rename executeJavaScript to something that describes the code functionality. This doesn't scale well if you have multiple JavaScript calls in the future. This could be renamed to something like "executeConsoleCode"